roxctl/common/auth/token_test.go

```diff
@@ -28,6 +31,22 @@ func (s *tokenSuite) Test_RetrieveAuthToken_WithEnv() {
 	s.Equal(got, testTokenVal, "Did not receive correct rox auth token from environment")
 }
 
+func (s *tokenSuite) Test_RetrieveAuthToken_WithFileEnv() {
+	tstDir, err := ioutils.TempDir("", "roxctl-test-common-auth")
+	s.NoError(err)
+	defer func() { _ = os.RemoveAll(tstDir) }()
```

Comment:
If you make the switch to `s.T().TempDir()` you won't require cleaning up, since the directory is cleaned up after the test is completed.